Week-4/JS_Fundamentals/JSArrays.js

```diff
@@ -0,0 +1,39 @@
+const num=prompt('Enter list of numbers separated by space : ');
+divideArray(num);
+function divideArray(num)
+{
+    const stringArray=num.split(' ');
+    const numArray=stringArray.map(item => parseInt(item));
+    let even=[];
+    let odd=[];
+
+    for(const x of numArray)
```

Comment:
avoid the use of for of loops
https://dev.to/jackshen/stop-using-for-loops-here-s-why-2hp